kamu/templates/rest_framework/api.html

```diff
@@ -0,0 +1,16 @@
+{% extends "rest_framework/base.html" %}
+{% load i18n %}
+
+{% block bootstrap_theme %}
+    <link rel="stylesheet" href="https://bootswatch.com/3/darkly/bootstrap.min.css" type="text/css">
```

Comment:
I would suggest saving this asset into our repo so that we can develop without relying on external resources :)

Reply:
Make sense, will do so. \o/